src/init.cpp

```diff
@@ -1728,12 +1714,20 @@ bool AppInitMain(InitInterfaces& interfaces)
     if (gArgs.IsArgSet("-blocknotify"))
         uiInterface.NotifyBlockTip_connect(BlockNotifyCallback);
 
-    std::vector<fs::path> vImportFiles;
-    for (const std::string& strFile : gArgs.GetArgs("-loadblock")) {
-        vImportFiles.push_back(strFile);
-    }
+    if (gArgs.IsArgSet("-loadblock")) {
+        const std::vector<std::string> vLoadBlockArgs = gArgs.GetArgs("-loadblock");
+
+        // Regard the '-noloadblock' argument
+        if (!vLoadBlockArgs.empty()) {
+            std::vector<fs::path> vImportFiles;
+            for (const std::string& strFile : vLoadBlockArgs) {
+                vImportFiles.push_back(AbsPathForConfigVal(strFile));
+            }
 
-    threadGroup.create_thread(std::bind(&ThreadImport, vImportFiles));
+            // Spawn 'loadblk' thread to run ThreadImport()
+            threadGroup.create_thread(std::bind(&ThreadImport, vImportFiles));
```

Comment:
The thread must be called currently (even if nothing is imported) to activate the best chain.

This explains the test failures, I believe

Reply:
That's interesting. This compiled and runs fine for me, but I guess since I'm already synced I didn't reproduce these test issues?

Reply:
Also, this is the only failure I see, but I assumed it was unrelated? `FAIL qt/test/test_bitcoin-qt (exit status: 134)`

Reply:
There's also this in appveyor but I wasn't totally sure what it meant:
`AssertionError: [node 0] Error: no RPC connection`